Sources/DataPipeline/CustomerIO.swift

```diff
@@ -21,6 +21,12 @@ public extension CustomerIO {
 
         let sdkConfig = SdkConfig.Factory.create(siteId: "", apiKey: "", region: .US)
         let newDiGraph = DIGraph(sdkConfig: sdkConfig)
+
+        // set the logLevel for ConsoleLogger
+        DIGraphShared.shared.logger.setLogLevel(logLevel)
+        // enable Analytics logs accordingly to logLevel
+        CustomerIO.shared.setDebugLogsEnabled(logLevel == .debug)
```

Comment:
Is there a reason for not putting this into `initialize` method? Keeping this outside `initialize` will result in `logLevel` not being set in `logger` for tests.

Reply:
[This seems to be in `initialize` method only. ](https://github.com/customerio/customerio-ios/blob/5d04ed4ef58736e54ab4b406992d4132df27044c/Sources/DataPipeline/CustomerIO.swift#L26)

Reply:
I didn't want to rely on `DiGraph` for this as well, as i would assume we decouple away from it.